vkcs/networking/data_source_sdn.go

```diff
@@ -0,0 +1,44 @@
+package networking
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/clients"
+	isdn "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/networking"
+)
+
+func DataSourceNetworkingSDN() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNetworkingSDNRead,
+
+		Schema: map[string]*schema.Schema{
+			"sdn": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "Names of available VKCS SDN's in the current project.",
+			},
+		},
+		Description: "Use this data source to get list of an available VKCS SDN in the current project.",
+	}
+}
+
+func dataSourceNetworkingSDNRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(clients.Config)
+	networkingClient, err := config.NetworkingV2Client(config.GetRegion(), isdn.SearchInAllSDNs)
+	if err != nil {
+		return diag.Errorf("Error creating VKCS networking client: %s", err)
+	}
+
+	sdn, err := isdn.GetAvailableSDNs(networkingClient)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	d.SetId(config.GetTenantID())
```

Comment:
Common way to do it is to set it to some random string, for example, string representation of unix time.

Reply:
I discussed this with @ftersin , we decided that it was better to set the project id

Reply:
it does not matter in my opinion

Reply:
I simply think that it isn't a good place to get tied to some authentication method, for example, there may be a case when we are authenticating through a token with no authentication scope set, and then the tenant ID will be empty.

Reply:
maybe then `config.GetTenantID() + hashcode.Strings(sdn)`?

Reply:
I don't see any reason to use tenant ID as id here, thus, it's just easier to stick to the common way:
https://github.com/vk-cs/terraform-provider-vkcs/blob/727f9b7c808891302bba7c75cf3d20645f19d550/vkcs/regions/data_source_regions.go#L58

Reply:
done